abstract/abstract.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'Geophysics in a changing climate'
+abstract: |
+ It is clear that the geosciences have an important role to play in working to understand and mitigate the societal impacts of the climate crisis. Locating critical minerals, monitoring geologic storage of CO2, managing groundwater, and characterizing changes to permafrost are all applications where geophysical data can provide insights. These applications raise interesting scientific questions about how to combine petrophysical, geologic, geochemical, and additional geophysical data sets to improve our ability to produce useful models of the subsurface. The next significant advancements will undoubtedly involve methodological improvements in inversions and machine learning, but importantly will require a more interdisciplinary approach, where the methods we design can be used to test and revise hypotheses specific to a given geologic context. The growth and adoption of open-source software and data standards is a positive step. Within the SimPEG community, we have seen many examples of how working in a modular, interoperable framework enables ideas to rapidly be prototyped and extended. Working in an open ecosystem allows us to more readily connect with methods in other fields, including geologic modelling and machine learning. 
+
+ Although geophysics has an important role to play in mitigating climate change and its impacts, we are at risk of continuing to be a “field in decline.” Geosciences as a whole have a severe diversity problem, and with the strong association of geophysics with hydrocarbons deterring many students, we are missing out on the talents and ideas of many. In this talk, I look forward to opening up what I hope will be continuing discussions about growing our community and broadening the scientific and societal contributions we can make. 
+---
+
+:::{iframe} https://www.youtube.com/watch?v=B6ZaiwcUaFA
+:width: 100%
+:::
```

Comment:
@rowanc1 or @fwkoch: could you help us with any pointers here? It looks like youtube doesn't like how we have tried to do the iframe 😞

Reply:
Give the YouTube embed link a try: https://www.youtube.com/embed/B6ZaiwcUaFA

Reply:
Yep. If you use the "share" button on youtube and then choose the embed option, copy the URL in the code into here. You can also choose the "privacy option" which seems better.

Reply:
Ah, thank you both! I had done that before, but it has been a while :) 

@tetikshajain, here are a couple of screenshot that show where this link can be found. From the "share" you click on "embed" 

![image](https://github.com/user-attachments/assets/e0ca7084-0459-4e2b-9c26-01320a6face0)

and it is this URL here

![image](https://github.com/user-attachments/assets/f961587a-6baf-4955-bdfb-1adce09f747e)


It also looks like you can just grab the id for the video from the share and then add it to `https://www.youtube.com/embed/<INSERT ID HERE>`

![image](https://github.com/user-attachments/assets/67e89d5b-d2bd-43d8-81a5-267e7ca4cc41)

Feel free to give it a try and we can see if that works. 

Thanks for your help @fwkoch and @rowanc1!!

Reply:
It seems to work now. Thank you!